lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -21,10 +21,21 @@ class BaseI18nTestCase(TestCase):
     """
     Base utilities for i18n test classes to derive from
     """
+    def __init__(self, *args, **kwargs):
+        super(BaseI18nTestCase, self).__init__(*args, **kwargs)
+        self.user = User()
+        self.client = Client()
+        # Url and site lang vars for tests to use
+        self.url = reverse('dashboard')
+        self.site_lang = settings.LANGUAGE_CODE
+        self.preview_language_url = '/update_lang/'
+        self.email = 'test@edx.org'
+        self.pwd = 'test_password'
```

Comment:
These all can be class variables on the test class instead.

And `self.client` should be created in the `setUp` method so it's reinitialized for each unit test.  Or maybe you don't need it since it's also created in the `user_login` method below.